scripts/createDataJson.js

```diff
@@ -0,0 +1,53 @@
+const program = require('commander');
+const fs = require('fs');
+const { sync: globSync } = require('glob');
+const last = require('lodash/last');
+
+let LANG_DIR = 'src/locales/';
```

Comment:
So for directories, I would prefer if we use the `path.resolve` or other `path` functions. It will generate absolute paths so we can be sure we get the right FS paths.